test/generator-create-api-mesh.test.js

```diff
@@ -30,13 +30,10 @@ describe('prototype', () => {
 })
 
 describe('plugin installed already', () => {
-  test('test that the generator invocation succeeds', async () => {
+  test('succeeds when plugin is installed and no mesh found', async () => {
     spawnCommandSpy
-      // list of plugins
```

Comment:
I know the comments are seemingly redundant here, but I actually find them helpful to know what each invocation of the spawnCommand is doing. Can you restore those for all tests?